crates/lib/src/qvm/mod.rs

```diff
@@ -11,9 +11,21 @@ use crate::RegisterData;
 
 mod execution;
 
-/// A map of register names (ie. "ro") to a [`RegisterData`] containing their values.
+/// Encapsulates data returned after running a program on the QVM
 #[allow(clippy::module_name_repetitions)]
-pub type QvmMemory = HashMap<String, RegisterData>;
+#[derive(Debug, Deserialize, Clone, PartialEq)]
+pub struct QvmResultData {
+    /// A map of register names (ie. "ro") to a [`RegisterData`] containing their values.
+    pub memory: HashMap<String, RegisterData>,
+}
```

Comment:
Was there a specific reason to move from a type alias to a wrapping struct? Just curious.

Reply:
Per [this recommendation](https://github.com/rigetti/qcs-sdk-rust/pull/223#discussion_r1092549376) to make sure we can accommodate any additional data the QVM may return in the future.